integrationtests/rtt_test.go

```diff
@@ -59,8 +64,22 @@ var _ = Describe("non-zero RTT", func() {
 
 		Context(fmt.Sprintf("with quic version %d", version), func() {
 			It("gets a file with 10ms RTT", func() {
-				runRTTTest(10*time.Millisecond, version)
+				runRTTTest(10*time.Millisecond, version, 20)
 			})
+
+			fileSizes := [...]int{dataLen, 5 * 1024 * 1024, dataLongLen}
+			for _, fileSize := range fileSizes {
+				fileSizeMB := fmt.Sprintf("%.2f", float64(fileSize)/1024/1024)
+				It("gets a "+fileSizeMB+"MB file with 75ms RTT", func() {
+					flags := log.Flags()
+					log.SetFlags(log.Ldate | log.Ltime | log.Lmicroseconds)
```

Comment:
Remove the 5 logging relate lines here and below – we don't always need the logs and they may skew the tests by slowing them down. We also already globally set the timestamp format for logs.